nerfstudio/models/nerfacto.py

```diff
@@ -352,6 +359,11 @@ def get_loss_dict(self, outputs, batch, metrics_dict=None):
                 loss_dict["pred_normal_loss"] = self.config.pred_normal_loss_mult * torch.mean(
                     outputs["rendered_pred_normal_loss"]
                 )
+
+            if "depth_image" in batch and self.step >= 500:
```

Comment:
Might want to get rid of the 500+ thing, it's a strange hyperparam (scheduling from 0 to 2000 or so is reasonable though)

Reply:
Made it scheduling from 0 to 2000 and looks like the results are good